Build issue in Angular 8 project (with resolution)

I have an Angular project using some ipfs related packages with cids as a dependency.  My ng build fails with the following error:

ERROR in ./node_modules/cids/src/index.js
Module not found: Error: Can't resolve 'multicodec/src/base-table'

Upon investigation, I found that cids has multicodec as a dependency, and attempts to import a json file using require without specifying the file suffix.  This results in the file not being identified and the aforementioned error.  By specifying 'multicodec/src/base-table.json' the issue can be bypassed.

I'm happy to submit a patch if you like.  I'd like to get this fixed in the upstream so I don't have to patch it locally in a postinstall step.  I believe it is not considered best practice to import json in this way, so I'd propose it be imported using fs.readFile/JSON.parse, but the remedy above works as well.

--- a/src/index.js
+++ b/src/index.js
@@ -3,7 +3,7 @@
 const mh = require('multihashes')
 const multibase = require('multibase')
 const multicodec = require('multicodec')
-const codecs = require('multicodec/src/base-table')
+const codecs = require('multicodec/src/base-table.json')
 const CIDUtil = require('./cid-util')
 const withIs = require('class-is')
 

